LICENSE.txt

```diff
@@ -0,0 +1,30 @@
+.. _license:
+
+LICENSE
+=======
+
+Copyright (c) 2015, MILA (Montreal Institute of Learning Algorithms)
```

Comment:
I amended the commit.

On Fri, Oct 16, 2015 at 12:59 PM, Pascal Lamblin <notifications@github.com>
wrote:

> In LICENSE.txt
> <https://github.com/mila-udem/summerschool2015/pull/37#discussion_r42264686>
> :
>
> > @@ -0,0 +1,30 @@
> > +.. _license:
> > +
> > +LICENSE
> > +=======
> > +
> > +Copyright (c) 2015, MILA (Montreal Institute of Learning Algorithms)
>
> Montreal Institute for Learning Algorithms
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mila-udem/summerschool2015/pull/37/files#r42264686>.
>